app/models/league.rb

```diff
@@ -63,6 +65,16 @@ def active_user_ranking
       .reorder('quota DESC')
   end
 
+  def team_ranking
+    teams
+    .select("teams.*, COUNT(matches.*) AS games_played, #{BASE_SCORE} + SUM(CASE WHEN matches.loser_team_id = teams.id THEN matches.difference * -1 ELSE matches.difference END) AS score")
```

Comment:
```CASE WHEN matches.loser_team_id = teams.id THEN matches.difference * -1```… meaning that if the team lost the match, it should lower the teams total score by the match score.